requirements/requirements.txt

```diff
@@ -5,6 +5,7 @@ h5py >= 2.8
 lmfit >= 1.0.1
 matplotlib >= 2.0
 mpmath >= 1.0.0
+numba
```

Comment:
Once you start adding tests for this, tox-minimal-37 will complain since it has its own list of dependencies. I'm adding that in `tox.ini` for you.

Reply:
I thought we already had `numba` for some reason, so I was surprised when all the tests started failing because of this.  Do you have any restrictions related to which `numba` version we should take?

Reply:
Yeah, we experimented with adding it in #953; at the time, Py3.9 was released, numba completely broke on it with its bytecode changes, and we didn't want to add version-dependent numba-or-nonumba logic. 0.53 is out now; things should be fine, as py39 will just (I think...) default to that as the only compatible dependency. Though we should look at the CI logs and check which runners get which version, just in case.

I don't have a clean lower bound on which numba versions are fine, though.